mappings/net/minecraft/world/gen/decorator/PredicatedDecorator.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_6661 net/minecraft/world/gen/decorator/PredicatedDecorator
+	METHOD method_38918 shouldDecorate (Lnet/minecraft/class_5444;Ljava/util/Random;Lnet/minecraft/class_2998;Lnet/minecraft/class_2338;)Z
```

Comment:
I don't have the decompiled code right now, but shouldn't this be called `shouldPlace`?